### ISSUE TITLE
Geonode 3 Installation using Docker error on loading main page

### ISSUE THREAD

**Issue by massoudhamad** [GeoNode/geonode#5986]
## Expected Behavior
The system should show the interface

## Actual Behavior
Is displaying the message as show in the picture bellow

"There was a problem loading this page
Please contact your GeoNode administrator (they may have received an email automatically if they configured it properly). If you are the site administrator, enable debug mode to see the actual error and fix it or file an issue in GeoNode's issue tracker"

## Steps to Reproduce the Problem

  1. output from docker-compose logs -f django
![Screenshot from 2020-04-09 10-18-42](https://user-images.githubusercontent.com/6734503/78868407-8f16e680-7a4b-11ea-8bbd-705af3b2a919.png)

django4geonode   | 1464 static files copied to '/mnt/volumes/statics/static'.
django4geonode   | static data refreshed
django4geonode   | Executing UWSGI server uwsgi --ini /usr/src/app/uwsgi.ini for Production
django4geonode   | command to be executed is uwsgi --ini /usr/src/app/uwsgi.ini
django4geonode   | [uWSGI] getting INI configuration from /usr/src/app/uwsgi.ini

  2.
  3.

## Specifications

  - GeoNode version: 3
  - Installation method (manual, GeoNode Docker, SPCGeoNode Docker): Geonode Docker
  - Platform: Ubuntu 18.04
  - Additional details:

**Comment by patlougheed** [comment]
Seeing the same issue. In the database container, the following errors are seen (via `docker logs db4geonode`):

```
2020-04-09 21:04:28.275 UTC [122] ERROR:  relation "base_region" does not exist at character 341
2020-04-09 21:04:28.275 UTC [122] STATEMENT:  SELECT "base_region"."id", "base_region"."code", "base_region"."name", "base_region"."name_en", "base_region"."parent_id", "base_region"."bbox_x0", "base_region"."bbox_x1", "base_region"."bbox_y0", "base_region"."bbox_y1", "base_region"."srid", "base_region"."lft", "base_region"."rght", "base_region"."tree_id", "base_region"."level" FROM "base_region" WHERE ("base_region"."level" = 0 OR "base_region"."parent_id" IS NULL) ORDER BY "base_region"."tree_id" ASC, "base_region"."lft" ASC
2020-04-09 21:05:51.896 UTC [134] ERROR:  relation "base_configuration" does not exist at character 109
2020-04-09 21:05:51.896 UTC [134] STATEMENT:  SELECT "base_configuration"."id", "base_configuration"."read_only", "base_configuration"."maintenance" FROM "base_configuration" WHERE "base_configuration"."id" = 1
2020-04-09 21:05:53.313 UTC [135] ERROR:  relation "base_configuration" does not exist at character 109
2020-04-09 21:05:53.313 UTC [135] STATEMENT:  SELECT "base_configuration"."id", "base_configuration"."read_only", "base_configuration"."maintenance" FROM "base_configuration" WHERE "base_configuration"."id" = 1
2020-04-09 21:05:54.268 UTC [136] ERROR:  relation "base_configuration" does not exist at character 109
2020-04-09 21:05:54.268 UTC [136] STATEMENT:  SELECT "base_configuration"."id", "base_configuration"."read_only", "base_configuration"."maintenance" FROM "base_configuration" WHERE "base_configuration"."id" = 1
2020-04-09 21:12:32.862 UTC [163] ERROR:  relation "base_configuration" does not exist at character 109
2020-04-09 21:12:32.862 UTC [163] STATEMENT:  SELECT "base_configuration"."id", "base_configuration"."read_only", "base_configuration"."maintenance" FROM "base_configuration" WHERE "base_configuration"."id" = 1
2020-04-09 21:12:32.918 UTC [163] ERROR:  relation "base_configuration" does not exist at character 109
2020-04-09 21:12:32.918 UTC [163] STATEMENT:  SELECT "base_configuration"."id", "base_configuration"."read_only", "base_configuration"."maintenance" FROM "base_configuration" WHERE "base_configuration"."id" = 1
```

**Comment by patlougheed** [comment]
Building the docker images from fresh appears to provide a fix, similarly to #3778. In particular:

```
docker-compose stop
docker rm <container ids>
docker rmi <image ids>
docker-compose -f docker-compose.yml -f docker-compose.override.localhost.yml build --no-cache
docker-compose -f docker-compose.yml -f docker-compose.override.localhost.yml up --build -d
```

Gives a (thus far) working GeoNode setup.

**Comment by massoudhamad** [comment]
Same as  here, I edited a docker-compose.override.localhost.yml and set Debug=True, then build it again solve the problem, 
docker-compose -f docker-compose.yml -f docker-compose.override.localhost.yml up --build -d
